docs/wagtail-pages.md

```diff
@@ -0,0 +1,211 @@
+that are rendered in a standard way. 
+All CFPB Wagtail pages should inherit from the 
+[`v1.models.base.CFGOVPage` class](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/base.py).
+
+!!! note
+    Before creating a new Wagtail page type 
+    please consider whether one of our existing page types can meet your needs, 
+    whether this is a different type of content than we have anywhere else in the site, 
+    and whether there is maintenance efficiency gained from a new page type.
+
+There are types of information defined on a new Wagtail page model: 
+basic [database fields](#fields) (like any Django model), 
+specialized database fields called [StreamFields](#streamfields) that allow for freeform page content, 
+and [editor panels](#panels) that present these fields to content editors. 
+
+## Fields
+
+Database fields in Wagtail pages work exactly the same as in 
+[Django models](https://docs.djangoproject.com/en/1.11/topics/db/models/#fields), 
+and Wagtail pages can use any [Django model field](https://docs.djangoproject.com/en/1.11/ref/models/fields/). 
+
+For example, our `BrowsePage` 
+[includes a standard Django `BooleanField`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py) 
+that allows content editors to toggle secondary navigation sibling pages:
+
+```python
+from django.db import models
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+```
+
+## StreamFields
+
+StreamFields are special Django model fields provided by Wagtail for 
+[freeform page content](https://docs.wagtail.io/en/v1.13.4/topics/streamfield.html). 
+They allow a content editor to pick any number number of optional components
+and place them in any order within their StreamField. 
+In practice this provides the flexibilty of a large 
+[rich text box with the structure of individual components](https://torchbox.com/blog/rich-text-fields-and-faster-horses/).
+
+For example, our `LandingPage` page model 
+[includes a `header` StreamField](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py) 
+that can have a hero and/or a text introduction:
+
+```python
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+```
+
+The specifics of StreamField block components can be found in 
+[Creating and Editing Wagtail Components](https://cfpb.github.io/cfgov-refresh/editing-components/).
+
+## Panels
+
+Editor panels define how the page's [fields](#fields) and [StreamFields](#streamfields) will be organized for content editors 
+and correspond to the tabs that appear across the top of the edit view for pages in the Wagtail admin.
```

Comment:
```suggestion
they correspond to the tabs that appear across the top of the edit view for a page in the Wagtail admin.
```

Reply:
This sentence kinda muddies the waters a bit... the tabs are not the panels; the tabs contain one or more panels. But it may not be important to make that distinction here.